kedro/io/kedro_data_catalog.py

```diff
@@ -112,6 +114,50 @@ def __eq__(self, other) -> bool:  # type: ignore[no-untyped-def]
             other.config_resolver.list_patterns(),
         )
 
+    def keys(self, regex_search: str | None = None) -> List[str]:  # noqa: UP006
+        return self._filter_keys(regex_search)
+
+    def values(self, regex_search: str | None = None) -> List[AbstractDataset]:  # noqa: UP006
```

Comment:
```suggestion
    def values(self, regex_search: str | None = None) -> list[AbstractDataset]:
```

Reply:
You can use `list` directly

Reply:
I had to change it to the current method because we have `list()` method defined for catalog and linter thinks you annotate using it rather than built-in `list`

![Screenshot 2024-10-09 at 11 11 43](https://github.com/user-attachments/assets/81c34420-9024-4b12-ab73-d6b2bdfb6d79)

<img width="713" alt="Screenshot 2024-10-09 at 11 14 43" src="https://github.com/user-attachments/assets/94c7f982-db96-4d60-b1e7-c201b5c5af63">